package/gluon-ddhcpd/luasrc/lib/gluon/upgrade/315-dhcp-config-firewall

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/lua
```

Comment:
WIth the mmfd rule it's probably appropriate to call the file `ddhcpd-firewall`.